guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>make_pipeline and make_column_transformer are provided by sklearn and we rely only on them to take preprocessors as input.</p><p> </p><p>Same for model.score() metrics are r2 score for regression and accuracy for classification.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>